CMakeLists.txt

```diff
@@ -424,6 +424,27 @@ if (KRATOS_SHARED_MEMORY_PARALLELIZATION STREQUAL "OpenMP")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
+
+    # Check if the environment variable OMP_SCHEDULE is defined
+    if(DEFINED ENV{OMP_SCHEDULE})
+        # Set the already defined one
+        set(KRATOS_OMP_SCHEDULE $ENV{OMP_SCHEDULE})
```

Comment:
`OMP_SCHEDULE` is a runtime env variable, it is a **extremely** bad idea to use it a compilation switch (IMO).

Reply:
I understand, but this is the following. 

During compilation the `OMP_SCHEDULE` will set `KRATOS_OMP_SCHEDULE` that will be used as default if actually `OMP_SCHEDULE` is not defined, but if `OMP_SCHEDULE` is defined `OMP_SCHEDULE` will be taken into account. Do you understand me?